Support for Firefox browser added

Nikitchenko opened the pull request:
HW task 1. CrossBrowser testing.

Commit:
Support for Firefox browser added

ivanmatiashepam commented on src/main/java/com/cucumber/junit/driver/DriverManager.java:
```diff
@@ -1,32 +1,50 @@
-        WebDriver driver = new ChromeDriver();
+    public static void setupDriver() {
+
+        WebDriver driver = null;
```
WebDriver instance is null by default - you can remove the assignment and leave just `WebDriver driver`

Nikitchenko replied:
done.

ivanmatiashepam commented on src/main/java/com/cucumber/junit/driver/DriverManager.java:
```diff
@@ -1,32 +1,50 @@
+            driver = new FirefoxDriver();
+        }
+        else {
+            throw new IllegalStateException("This driver is not supported: " + System.getProperty("browser"));
```
Please use the existing "browser" String variable, no need to execute getProperty() one more time

Nikitchenko replied:
done

Commit:
Slight refactoring
(cleaning code for DriverManager,
 update POM for add support of mvn test and avaid warnings)

Commit:
microfix

Commit:
change to firefox

Commit:
test data updated
